pkg/commands/domainv1/tools/status_test.go

```diff
@@ -0,0 +1,129 @@
+package tools_test
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+	"testing"
+
+	root "github.com/fastly/cli/pkg/commands/domainv1"
+	"github.com/fastly/cli/pkg/commands/domainv1/tools"
+	"github.com/fastly/cli/pkg/testutil"
+	"github.com/fastly/go-fastly/v10/fastly"
+	"github.com/fastly/go-fastly/v10/fastly/domains/v1/tools/status"
+)
+
+func TestNewDomainsV1ToolsStatusCommand(t *testing.T) {
+	scenarios := []testutil.CLIScenario{
+		{
+			Args:      "",
+			WantError: "error parsing arguments: required flag --domain not provided",
+		},
+		{
+			Args: "--domain domainr-testing.com",
```

Comment:
As this is a Fastly product and Fastly API endpoint, we should probably use a non-domainr domain.